Components/Map.js

```diff
@@ -77,10 +82,14 @@ export default class Map extends React.Component {
     console.log("map rendered");
     return (
       <MapView
+        customMapStyle={desaturatedSubtleMapStyle}
```

Comment:
Please switch around `desaturatedSubtleMapStyle` to `subtleMapStyle`, `silverMapStyle`, `darkMapStyle` to see what is most fitting.

Reply:
subtleMapStyle!